topcoffea/modules/dataDrivenEstimation.py

```diff
@@ -109,7 +109,15 @@ def DDFakes(self):
                         
                         # now we take all the stuff that is not data in the AR to make the prompt subtraction and assign them to nonprompt.
                         hPromptSub=hAR.group('sample', hist.Cat('sample','sample'), newNameDictNoData )
-                        
+
+                        # remove the up/down variations (if any) from the prompt subtraction histo
+                        syst_var_idet_rm_lst = []
+                        syst_var_idet_lst = hPromptSub.identifiers("systematic")
+                        for syst_var_idet in syst_var_idet_lst:
+                            if syst_var_idet.name != "nominal":
+                                syst_var_idet_rm_lst.append(syst_var_idet)
+                        hPromptSub = hPromptSub.remove(syst_var_idet_rm_lst,"systematic")
+
```

Comment:
Would
```suggestion
                       hPromptSub = hPromptSub.integrate("systematic", "nominal")
```
be simpler, or is there a good reason to avoid integrating over the axis?

Reply:
I suppose my suggestion would also remove `"nominal"` from the axis, so maybe it's not a good idea.

Reply:
Right, so using `integrate` to get the `nominal` category would remove the `systematic` axis from the histogram. I think we want to keep this axis, but only keep the `nominal` category. If this were the only axis, I think we could just do `histo["nominal"]`, but since there are multiple axes in this hist, that does not work. Anyway, there may be a way to keep just the `nominal` category of the `systemaitc` axis, but I was not not sure how, so it seemed like just removing the unwanted categories would be an ok workaround.